packages/validator-ajv8/src/validator.ts

```diff
@@ -148,7 +151,7 @@ export default class AJV8Validator<T = any, S extends StrictRJSFSchema = RJSFSch
         return { ...acc, [key]: this.createErrorHandler(value) };
       }, handler);
     }
-    if (isObject(formData)) {
+    if (isPlainObject(formData)) {
```

Comment:
Not clear to me why we need to be more restrictive with `isPlainObject` instead of `isObject`, but I don't think it will be an issue